src/modules/flow/platform/platform.json

```diff
@@ -36,6 +36,22 @@
       "url": "http://solettaproject.org/doc/latest/node_types/platform.html"
     },
     {
+      "category": "input/sw",
+      "description": "Node used to set the platform target",
+      "in_ports": [
+        {
+          "data_type": "string",
+          "description": "The new platform target to set which must be one of: default, rescue, emergency, poweroff, reboot or suspend.",
+          "methods": {
+            "process": "platform_target_process"
+          },
+          "name": "TARGET"
+        }
+      ],
+      "name": "platform/target",
```

Comment:
Currently there's no API to get the current target. Maybe we can create one if possible.
For now I will rename the node to set-target and create a sample for it.